sw/device/lib/dif/dif_kmac.c

```diff
@@ -775,6 +775,7 @@ dif_result_t dif_kmac_config_is_locked(const dif_kmac_t *kmac,
   uint32_t reg =
       mmio_region_read32(kmac->base_addr, KMAC_CFG_REGWEN_REG_OFFSET);
   *is_locked = bitfield_bit32_read(reg, KMAC_CFG_REGWEN_EN_BIT);
```

Comment:
Maybe 
```suggestion
  *is_locked = !bitfield_bit32_read(reg, KMAC_CFG_REGWEN_EN_BIT);
```
to be concistent with the other DIFs like [here](https://github.com/lowRISC/opentitan/blob/5db82833a4e9207e59b44236947b4548156f947b/sw/device/lib/dif/dif_rstmgr.c#L74)

Reply:
Thanks @engdoreis, I updated it accordingly.